Android/ThirdPartyAuth/app/src/main/java/com/bentley/sample/thirdpartyauth/ThirdPartyAuthApplication.kt

```diff
@@ -0,0 +1,31 @@
+package com.bentley.sample.thirdpartyauth
+
+import android.app.Application
+import android.content.Context
+import com.bentley.sample.shared.MainActivity
+
+class ThirdPartyAuthApplication : Application() {
+
+	lateinit var loginViewModel: LoginViewModel
```

Comment:
This file appears to be using tabs instead of spaces for indentation. Some of the files in this commit definitely use 4 spaces, and some use tabs. All the files using tabs need to switch to using 4 spaces. I would strongly recommend changing the tabs to spaces in an isolated commit that has no other changes.

Reply:
Done. I created a project-level formatting file to do this all automatically. I'm not sure there's a way to automatically incorporate it into all the projects, but it might be worth centralizing it in the Android/Shared folder and importing it into each sample.